tls/s2n_ktls_io.c

```diff
@@ -13,9 +13,24 @@
  * permissions and limitations under the License.
  */
 
+#if defined(__FreeBSD__) || defined(__APPLE__)
+    /* FreeBSD and APPLE require POSIX compatibility off to access CMSG_* macros.
+     * On FreeBSD this enables __BSD_VISIBLE.
+     */
+    #undef _POSIX_C_SOURCE
+#endif
```

Comment:
Apple source for sys/socket.h

https://gist.github.com/toidiu/0b63f753c2b0ab3b3ffbd77efee6e6eb